<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PDF View</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_main">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_docs">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_noDocs">
            <property name="text">
             <string>No Docs</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_pages">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_noPages">
            <property name="text">
             <string>No Pages</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="GraphicsView" name="graphicsView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_console">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Console</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>757</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdvanced">
    <property name="title">
     <string>Advanced</string>
    </property>
    <addaction name="action_Debug_Console"/>
   </widget>
   <addaction name="menuAdvanced"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Save_Session"/>
   <addaction name="action_Open_Session"/>
   <addaction name="separator"/>
   <addaction name="action_Add_Document"/>
   <addaction name="separator"/>
   <addaction name="action_Crop"/>
   <addaction name="separator"/>
   <addaction name="action_Previous_Page"/>
   <addaction name="action_Next_Page"/>
  </widget>
  <action name="action_Debug_Console">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Debug Console</string>
   </property>
  </action>
  <action name="action_Add_Document">
   <property name="text">
    <string>Add Document</string>
   </property>
  </action>
  <action name="action_Save_Session">
   <property name="text">
    <string>Save Session</string>
   </property>
  </action>
  <action name="action_Open_Session">
   <property name="text">
    <string>Open Session</string>
   </property>
  </action>
  <action name="action_Next_Page">
   <property name="text">
    <string>Next Page</string>
   </property>
  </action>
  <action name="action_Previous_Page">
   <property name="text">
    <string>Previous Page</string>
   </property>
  </action>
  <action name="action_Crop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Crop</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>graphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
